<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pose2Sim Processes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibration</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sessionFolderText">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="poseEstimationButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Pose Estimation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="synchronizationButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Synchronization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="personAssociationButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Person Association</string>
    </property>
   </widget>
   <widget class="QPushButton" name="triangulationButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filteringButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtering</string>
    </property>
   </widget>
   <widget class="QPushButton" name="markerAugmentationButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker Augmentation</string>
    </property>
   </widget>
   <widget class="QLabel" name="sessionFolderLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Session Folder:</string>
    </property>
   </widget>
   <widget class="QLabel" name="trialFolderLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trial Folder:</string>
    </property>
   </widget>
   <widget class="QLabel" name="trialFolderText">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="threadCheckButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Thread Check</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
